htmd/parameterization/cli.py

```diff
@@ -106,6 +107,13 @@ def _prepare_molecule(args):
 
     # TODO: check bonds
 
+    # Detect chiral centers
+    mol.chiral_centers = detectChiralCenters(mol)[0]
```

Comment:
I'm not a huge fan of storing arbitrary values inside the Molecule class. Would it be much uglier to keep the chiral centers around as a variable?

Reply:
Cluttering the high-level code with low-level variables is also not good. On the other hand, chirallity is an intrinsic property of a molecule. What do you propose?

Reply:
What you are doing here is storing volatile data inside the class. It is a volatile property that depends on changes to the coordinates so it should not be stored, but calculated on request instead. I would be fine for example if there was a `mol.detectChiralCenters()` method but I would not store the data in the class. 

So the data should be passed around as a low-level variable yes, since it's only used to compare an old state of the molecule to a new state of the molecule.

Reply:
All other properties we store in the Molecule are non-volatile. You can argue that bonds and charges are volatile in QM but HTMD is built for MD so we assume that they are not and thus bonds, dihedrals, angles, charges etc can be stored inside the class without ever changing